libraries/ESP8266EthernetWebServer/library.properties

```diff
@@ -0,0 +1,9 @@
+name=ESP8266EthernetWebServer
+version=1.0
+author=Ivan Grokhotkov
+maintainer=Ivan Grokhtkov <ivan@esp8266.com>
```

Comment:
I like your idea. Reason why I just fork that library instead doing generics was that I am not so good with c++. Because I had to get ethernet support I just fork it. If you can make support to that original library it would be even better. What I hope is that you also take this pin documentation etc from sample sketch and add it to wiki documentation. I use total 7 full days in last 6 months to hit my head to wall because of undocumented Ethernet library. I finally get it work and I hope that no one else need to feel my pain and do so much try & errors if A) pins are documented B) know that it can take almost minute that tcp server wake up even it reply already to ping.